draft/implementation-notes/index.html

```diff
@@ -151,42 +151,43 @@ <h2>Filesystem metadata</h2>
 
 <h2>Empty Directories</h2>
 <p>
-OCFL preserves files and their content, with directories serving as a useful organisational convention. An empty
+OCFL preserves files and their content, with directories serving as a useful organizational convention. An empty
 directory consists only of filesystem metadata and therefore, as noted above, is not amenable to direct
 preservation in OCFL. If the preservation of empty directories is considered essential then the suggested route
-is to insert a zero length file named <code>.keep</code> into the directory which will thus be preserved as
-part of the files' path.
+is to insert a zero length file named <code>.keep</code> into the directory which will ensure directories are preserved
+as part of the files' path.
```

Comment:
should be **file's** instead of **files'** as we are talking about only one file ("a zero length file")